.github/workflows/build-test.yml

```diff
@@ -3,12 +3,10 @@ on:
    push:
       branches:
         - main
-        - test
    pull_request:
       branches:
         - main
-        - test
-
+   workflow_dispatch:
```

Comment:
Why does `workflow_dispatch` have no attributes/values/config?

Reply:
Dunno,  just clone code Jake wrote to turn on the "run workflow" button elsewhere.

Reply:
OK, sounds good.